trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/CST/create_airfoil_from_CST.py

```diff
@@ -0,0 +1,137 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil-CST
```

Comment:
See previous files comments

Reply:
How are these substantially different?

Reply:
The CST definition is easier to work with if one considers optimization and one of the design variables is the airfoil shape

Reply:
I have made minor changes to the Oswald efficiency, but the regression fails B737 (which makes sense) what should I do?

Reply:
One possible option is to allow both methods to exist and use a flag in the analysis setting that determines which is used. You would then need to check both of them.